Data/ListTrie/Base/Map.hs

```diff
@@ -15,9 +15,11 @@ import Data.Function       (on)
 import Data.List           (foldl1', mapAccumL, nubBy, partition, sort, sortBy)
 import Data.Ord            (comparing)
 import Data.Traversable    (Traversable(..), mapAccumR)
-import qualified Data.IntMap as IM
-import qualified Data.Map    as M
-
+import qualified Data.IntMap     as IM
+import qualified Data.Map        as M
+#if MIN_VERSION_containers(0,5,0)
+import qualified Data.Map.Strict as M.Strict
```

Comment:
... huh, seems I was completely mistaken. I thought that `Data.Map.Strict.Map` and `Data.Map.Lazy.Map` were actually different types, but apparently not. So much for my rambling in #7, sorry for needlessly delaying this.